tests/AppTest.cpp

```diff
@@ -339,7 +339,7 @@ TEST_F(TApp, doubleVectorFunction) {
     });
     args = {"--val", "5", "--val", "6", "--val", "7"};
     run();
-    EXPECT_EQ(res.size(), 3);
+    EXPECT_EQ(res.size(), (size_t)3);
```

Comment:
I think `EXPECT_EQ(res.size(), 3u);` is better here :)

Reply:
Okay, this can be changed now. I've converted to using `#u` everywhere in testing. Thanks!